### ISSUE TITLE
Cannot get Emmet to work Mac

### ISSUE THREAD

**Issue by guylepage3** [sergeche/emmet-sublime#669]
Cannot get Emmet to work.

**Environment**
- MacBook Pro (15-inch, 2017)
- macOS Sierra Version 10.12.6 (16G29)
- Sublime Text 3 (3143)

Installed with Package Control in Sublime Text 3.

**Comment by lv-code** [comment]
I have the issue too.

**Comment by sergeche** [comment]
Do you see Emmet commands in Commands Pallette?

**Comment by lv-code** [comment]
no, hove not Command like 'Emmet' in the Commands Pallette, Is that to say the Emmet not install successfully?

**Comment by sergeche** [comment]
Yes, it means Emmet is either not installed or it failed to start. Check out ST console for errors

**Comment by Kon-Xen** [comment]
Hi I have a  similar problem.
osX El Capitan, Sublime 3.
Emmet show in the installed and activated lists (package control)
but it doesn't appear to be working,
Im typing
  li>a 
and nothing happens :(
is it because of the aggigg osx?
